### PR TITLE
✨ [Amp story panning media] Validation

### LINKED ISSUE #31506: [Panning media] validation
Validation for attributes and element child. 

cc @ampproject/wg-stories

### PR DESCRIPTION
**Pull request by processprocess**
Validation for attributes and amp-img child.
Update demo template.

Fixes #31506

### CODE REVIEW

File: extensions/amp-story-panning-media/validator-amp-story-panning-media.protoascii [hunk 1]
```diff
@@ -17,4 +17,29 @@ tags: {  # <amp-story-panning-media>
...
+    value_regex: "-?\\d+\\.?\\d*(%)"
+    trigger: {
+      also_requires_attr: "duration"
+    }
+  }
+  attrs: {
+    name: "y"
+    value_regex: "-?\\d+\\.?\\d*(%)"
+    trigger: {
+      also_requires_attr: "duration"
```
Comment (gmajoulet) [inline extensions/amp-story-panning-media/validator-amp-story-panning-media.protoascii:31]: How does this work? I don't see the `duration` in your test cases nor in this proto, does it need be removed?

Comment (processprocess) [reply extensions/amp-story-panning-media/validator-amp-story-panning-media.protoascii]: It does yes. Thank you for catching this.

**Review by gmajoulet — APPROVED** [review]

File: extensions/amp-story-panning-media/validator-amp-story-panning-media.protoascii [hunk 2]
```diff
@@ -17,4 +17,23 @@ tags: {  # <amp-story-panning-media>
   amp_layout: {
     supported_layouts: FILL
   }
+  attrs: {
+    name: "x"
```
Comment (Gregable) [inline extensions/amp-story-panning-media/validator-amp-story-panning-media.protoascii:21]: have you considered using `data-` attributes in order to reduce the possibility of a collision with a browser attribute in the future? Short attributes like this seem more likely than average to cause problems down the line.

Comment (processprocess) [reply extensions/amp-story-panning-media/validator-amp-story-panning-media.protoascii]: Thanks for mentioning this Greg! Agreed that it's a good idea to add them. Updated.

**Review by twifkak — APPROVED** [review]

File: extensions/amp-story-panning-media/validator-amp-story-panning-media.protoascii [hunk 3]
```diff
@@ -17,4 +17,23 @@ tags: {  # <amp-story-panning-media>
...
+  attrs: {
+    name: "data-y"
+    value_regex: "-?\\d+\\.?\\d*(%)"
+  }
+  attrs: {
+    name: "data-zoom"
+    value_regex: "\\d+\\.?\\d*"
+  }
+  attrs: {
+    name: "lock-bounds"
```
Comment (Gregable) [inline extensions/amp-story-panning-media/validator-amp-story-panning-media.protoascii:33]: It looks like you intend for this to have no value, a presence/missing attr. As written, you allow *any* value. To have no value, specify:

```
value: ""
```

Comment (processprocess) [reply extensions/amp-story-panning-media/validator-amp-story-panning-media.protoascii:33]: Thank you Greg!

**Pull request merged**